pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -186,7 +186,7 @@ type ContainerState struct {
 }
 
 type NetworkSettings struct {
-	Ports *nat.PortMap `json:",omitempty"`
+	Ports *nat.PortMap
```

Comment:
Docker seems to use `Ports nat.PortMap`
https://github.com/moby/moby/blob/v27.1.2/api/types/types.go#L302